curl-config.c.in

```diff
@@ -0,0 +1,429 @@
+/***************************************************************************
+ *                                  _   _ ____  _
+ *  Project                     ___| | | |  _ \| |
+ *                             / __| | | | |_) | |
+ *                            | (__| |_| |  _ <| |___
+ *                             \___|\___/|_| \_\_____|
+ *
+ * Copyright (C) Daniel Stenberg, <daniel@haxx.se>, et al.
+ *
+ * This software is licensed as described in the file COPYING, which
+ * you should have received as part of this distribution. The terms
+ * are also available at https://curl.se/docs/copyright.html.
+ *
+ * You may opt to use, copy, modify, merge, publish, distribute and/or sell
+ * copies of the Software, and permit persons to whom the Software is
+ * furnished to do so, under the terms of the COPYING file.
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTY OF ANY
+ * KIND, either express or implied.
+ *
+ * SPDX-License-Identifier: curl
+ *
+ ***************************************************************************/
+
+#include <ctype.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+/* install prefix */
+/*
+  FIXME: if someone uses --prefix with cmake --install, this will not match
+  the actual installation directory. should run a CMake script on install to
+  reconfigure and recompile as necessary in order to ensure coherency. note
+  tht the pkg-config file generated will have the same problem.
+*/
+static const char *prefix = "@prefix@";  /* CMAKE_INSTALL_PREFIX */
+/* exec prefix (fully expanded). we don't use this however */
+/* static const char *exec_prefix = "@lit_exec_prefix@"; */
+/* pkg-config C/C++ flags */
+static const char *cppflag_curl_staticlib = "@LIBCURL_PC_CFLAGS@";
```

Comment:
I kept the variable names similar for the most part to make it easier to compare against `curl-config.in`.